OpenRA.Mods.Common/UtilityCommands/CheckImageReferences.cs

```diff
@@ -41,6 +41,25 @@ void IUtilityCommand.Run(Utility utility, string[] args)
 				var nodes = MiniYaml.Merge(modData.Manifest.Sequences.Select(s => MiniYaml.FromStream(modData.DefaultFileSystem.Open(s), s)));
 				foreach (var n in nodes.Where(node => !node.Key.StartsWith(ActorInfo.AbstractActorPrefix, StringComparison.Ordinal)))
 					modData.SpriteSequenceLoader.ParseSequences(modData, ts, sc, n);
+
+				foreach (var template in ts.Templates)
+				{
+					foreach (var image in template.Value.Images)
+					{
+						if (!modData.DefaultFileSystem.Exists(image))
+						{
+							failed = true;
+							Console.WriteLine("\t" + "Tileset file {0} not found.".F(image));
+						}
+
+						var sprites = sc[image];
```

Comment:
This crashes if the image isn't found.

Reply:
It seems strange to me that we would crash with a useful exception in this case, but silently continue without crashing in the case above. We could potentially follow the lead of `modData.SpriteSequenceLoader.OnMissingSpriteError` for tilesets too?

Reply:
Added the `continue` to stop that.